test/js/bun/util/inspect.test.js

```
@@ -82,6 +82,30 @@ it("getter/setters", () => {
+      set(target, key, value, receiver) {
+        return Reflect.set(target, key, value, receiver);
+      },
+    };
+    originalPrototype = Object.getPrototypeOf(value);
+    newPrototype = new Proxy(originalPrototype, handler);
+    Object.setPrototypeOf(value, newPrototype);
+  }
+  class Foo {
+    get bar() {
+      Bun.inspect(this);
+    }
```

Biome (2.1.2)

[error] This getter should return a value.



(lint/suspicious/useGetterReturn)